app/utils/repo-actions-item-component-mixin.js

```diff
@@ -21,18 +25,36 @@ export default Ember.Mixin.create({
     return this.get('item.canRestart') && this.get('userHasPermissionForRepo');
   }.property('userHasPermissionForRepo', 'item.canRestart'),
 
+  displayFlashError(status) {
+    let type = this.get('type');
+    if (status === 422 || status === 400) {
+      this.get('flashes').error(`This ${type} can't be cancelled`);
```

Comment:
Is this because the `/restart` endpoint won't return an error code currently?  Can we simply not invoke the function in that case, or is there something else that should be done?